common/cache/cache_test.go

```diff
@@ -0,0 +1,133 @@
+package cache
+
+import (
+	"testing"
+)
+
+func TestNewLRUCache(t *testing.T) {
+	lruCache := New(5)
+	lruCache.Add("hello", "world")
+	c := lruCache.Contains("hello")
+	if !c {
+		t.Fatal("expected cache to contain \"hello\" key")
+	}
+
+	v := lruCache.Get("hello")
+	if v == nil {
+		t.Fatal("expected cache to contain \"hello\" key")
+	}
+	if v.(string) != "world" {
+		t.Fatal("expected \"hello\" key to contain value \"world\"")
+	}
+
+	r := lruCache.Remove("hello")
+	if !r {
+		t.Fatal("expected \"hello\" key to be removed from cache")
+	}
+
+	v = lruCache.Get("hello")
+	if v != nil {
+		t.Fatal("expected cache to not contain \"hello\" key")
+	}
+}
+
+func TestNewLRUCache_ContainsOrAdd(t *testing.T) {
```

Comment:
A similarly minor one `TestContainsOrAdd` would be enough so we don't need underscores

Reply:
Default auto complete names of goland like to do this was lazy